ClientApp/src/types/task.ts

```diff
@@ -3,4 +3,8 @@ export type TaskData = {
   name: string;
   assignee: string;
   storyPoints: number;
+  description: string;
```

Comment:
What are your thoughts on adding more fields to the task data type now vs waiting for more implementation? I'm fine merging now and adding later, but it does mean that another PR will have to address fields like priority, cost, creation approval.

Reply:
Yeah I see what you mean, but keeping it minimal for now is cleaner in the code while we aren't ready to digest the backend api yet, especially if not everything is implemented on the pages yet either. Otherwise I'd have to stick fake data in the server-api for every one of those fields, which would be a lot.